packages/alchemy/src/react/components/auth/card/add-passkey.tsx

```diff
@@ -13,34 +39,57 @@ export const AddPasskey = () => {
     },
   });
 
+  const renderRow = useCallback(
+    ({
+      icon,
+      title,
+      description,
+    }: {
+      icon: ReactNode;
+      title: string;
+      description: string;
+    }) => {
+      return (
+        <div key={title} className="flex gap-2">
+          <div className="h-5 w-5 flex items-center justify-center">{icon}</div>
+          <div className="flex flex-col">
+            <p className="font-semibold text-sm">{title}</p>
+            <p className="text-fg-secondary text-sm">{description}</p>
+          </div>
+        </div>
+      );
+    },
+    []
+  );
```

Comment:
any reason not to pull this out into its own component? seeing it's just a `useCallback` with empty deps it may as well be a static component

Reply:
makes sense to me! I'll do that in the next PR